### PR TITLE
add plot util for getting voltage angle 

### PR DESCRIPTION
**Pull request by m-bossart**

### CODE REVIEW

File: src/PowerSimulationsDynamics.jl [hunk 1]
```diff
@@ -19,6 +19,7 @@ export MassMatrixModel
 export small_signal_analysis
 export get_state_series
 export get_voltagemag_series
```
Comment (jd-lara) [inline src/PowerSimulationsDynamics.jl:21]: We should also spell `voltage_magnitude`

**Review by jd-lara — CHANGES_REQUESTED** [review]
Can you make the requested name changes to the function names and format correctly the code please?

**Comment by m-bossart** [comment]
so the functions should be named `get_voltage_magnitude_series` and `get_voltage_angle_series`?

**Comment by jd-lara** [comment]
> so the functions should be named `get_voltage_magnitude_series` and `get_voltage_angle_series`?

Yes please

File: test/test_case01_OMIB.jl [hunk 2]
```diff
@@ -46,7 +46,7 @@ Ybus_change = NetworkSwitch(
         δ = series[2]
         #Clean Extra Point at t = 1.0 from Callback
 
-        series2 = get_voltagemag_series(sim, 102)
+        series2 = get_voltage_magnitude_series(sim, 102)
```
Comment (rodrigomha) [inline test/test_case01_OMIB.jl:49]: Can you test the function? You can add `series3 = get_voltage_angle_series(sim, 102)`.

Comment (rodrigomha) [reply test/test_case01_OMIB.jl:49]: Only do it in Test01

Comment (jd-lara) [inline test/test_case01_OMIB.jl:49]:
```suggestion
        series2 = get_voltage_magnitude_series(sim, 102)
        series3 = get_voltage_angle_series(sim, 102)
```

**Pull request merged**